src/main.py

```diff
@@ -365,6 +365,7 @@ def main_loop():
 
 
 def main():
+    should_loop = not str_to_bool(os.getenv("RUN_ONLY_ONCE", "False"))

```

Comment:
This should be called run_only_once and with the changes below we can remove the not so its simpler to understand.